src/middlewared/middlewared/schema.py

```diff
@@ -555,3 +555,35 @@ def nf(*args, **kwargs):
 
         return nf
     return wrap
+
+
+class Unix(Attribute):
```

Comment:
Please rename this to `UnixPermissions` or something like that. Just `Unix` is very ambiguous.